seed/ts-sdk/no-environment/README.md

```diff
@@ -101,6 +101,18 @@ const response = await client.dummy.getDummy(..., {
 controller.abort(); // aborts the request
 ```
 
+### Raw Responses
+
+The SDK provides access to raw response data, including headers, through the `.withRawResponse()` method.
+The `.withRawResponse()` method returns a promise that results to an object with a `data` and a `rawResponse` property.
+
+```typescript
+const { data, rawResponse } = await client.dummy.getDummy(...).withRawResponse();
+
+console.log(data);
+console.log(rawResponse.headers['X-My-Header']);
+```
```

Comment:
Thoughts on putting this in the `advanced` section? We could also title this something like `Access Raw Response Data`.

Reply:
It is under advanced. I'll update the title